weblab/conftest.py

```diff
@@ -26,13 +26,14 @@ def add_version(entity,
                     tag_name=None,
                     visibility=None,
                     cache=True,
+                    message='file',
```

Comment:
would something like 'Add file' make more sense as a typical commit message. Or does the single term 'file' have signifiance I'm missing?

Reply:
It's only for tests, so doesn't really matter what it is. Some tests probably assume 'file' now.